projects/woff2/Dockerfile

```diff
@@ -20,4 +20,5 @@ RUN apt-get install -y make autoconf automake libtool
 
 RUN git clone --depth 1 --recursive https://github.com/google/woff2
 WORKDIR woff2
-COPY build.sh convert_woff2ttf_fuzzer.* $SRC/
+COPY build.sh *.options seed_corpus.zip $SRC/
```

Comment:
Oops, github doesn't show what I highlighted. I meant `seed_corpus.zip`. I don't see where we download it or anything else.